tests/unit/test_grpc_helpers.py

```diff
@@ -365,52 +365,92 @@ def test_wrap_errors_streaming(wrap_stream_errors):
     wrap_stream_errors.assert_called_once_with(callable_)
 
 
-@mock.patch("grpc.composite_channel_credentials")
+@pytest.mark.parametrize(
```

Comment:
Would it be worth making this test function take a target parameter, and then parametrizing it as we do for `test_create_channel_implicit_with_default_host` below? `target` seems to be used the same way in both places, and this would allow us to check here the `dns:///` and `another-c2p:///` cases you parametrize below.

Reply:
Done in https://github.com/googleapis/python-api-core/pull/583/commits/396d8fa364daacbcfa5eafcb2b6c51f02504995a